web/rest/platform/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 2.17.0
+* Models:
+    - Company
+        - Set country as required property
+    - DDI
+      - Removed country from required properties
```

Comment:
I thought DDI creation was part of Brand API and Platform could only read values (where required may not apply)